README.md

```diff
@@ -26,6 +26,18 @@ Read more about Xash3D on ModDB: https://www.moddb.com/engines/xash3d-engine
 * Voice support
 * Vulkan renderer
 
+## Installation & Running
+0) Download Xash3D binaries: you can use [release](https://github.com/FWGS/xash3d-fwgs/releases) build (recommended) or [testing](https://github.com/FWGS/xash3d-deploy/tree/anewengine-master) build, also you can compile engine from sources.
+Choose proper build package depending on which platform you're using.
+1) Copy engine binaries to some directory.
+2) Copy `valve` directory from [Half-Life](https://store.steampowered.com/app/70/HalfLife/) to mentioned above directory with engine binaries.
+Also if you're using Windows: you should copy `SDL2.dll` and `vgui.dll` libraries from Half-Life directory to Xash3D directory.
```

Comment:
We distribute our own SDL2.dll.

Reply:
It may be not a good solution if Valve games provides SDL2 and linux distributives too.

Reply:
@nekonomicon naah.

SDL2 keeps binary compatibility. At least, it should.

The only thing, we can't go backwards and there isn't too much sense doing that.